src/routes/articles/create-article.e2e.test.js

```diff
@@ -14,26 +14,6 @@ describe('POST /articles', () => {
 		)
```

Comment:
Kindly update any other part of this test that is sending `userId` as part of the request body

Reply:
sure

Reply:
I have implemented the necessary changes